MonoGame.Framework/Graphics/OpenGL.cs

```diff
@@ -1522,7 +1522,7 @@ protected unsafe static IntPtr MarshalStringToPtr (string str)
             if (intPtr == IntPtr.Zero) {
                 throw new OutOfMemoryException ();
             }
-            fixed (char* chars = str + RuntimeHelpers.OffsetToStringData / 2) {
+            fixed (char* chars = str) {
```

Comment:
The original code was actually doing something like this:

`fixed (char* chars = String.Concat(str, (RuntimeHelpers.OffsetToStringData / 2).ToString()))`